Lib/test/test_logging.py

```diff
@@ -4406,6 +4406,29 @@ def test_basic(self):
         self.assertTrue(found, msg=msg)
 
 
+class CachingTest(BaseTest):
+    def test_caching(self):
+        root = self.root_logger
+        level1 = logging.getLogger("abc")
+        level2 = logging.getLogger("abc.def")
+
+        root.setLevel(logging.ERROR)
+        self.assertEqual(level2._cache, {})  # Cache is empty
+
+        self.assertTrue(level2.isEnabledFor(logging.ERROR))
+        self.assertEqual(level2._cache, {logging.ERROR: True})  # Cache is populated
```

Comment:
What values would the logger level cache dict have? Either empty or a level pointing to `True`? Could there be multiple keys in this dict?